packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -114,6 +141,14 @@ export default function BlockToolbar( { hideDragHandle } ) {
 				{ ( shouldShowVisualToolbar || isMultiToolbar ) && (
 					<ToolbarGroup className="block-editor-block-toolbar__block-controls">
 						<BlockSwitcher clientIds={ blockClientIds } />
+						{ isMultiToolbar && (
```

Comment:
It could be worth moving this slightly further down into its own `ToolbarGroup` to the right (visually) of the `BlockMover`, so that it's in its own section, in case we'd like to add other similar controls in the future?

Reply:
Yes, this should be present in the actual block tools, not in the block type meta functions (transforms, movers, etc).